sonic_data_client/db_client.go

```diff
@@ -23,8 +23,8 @@ import (
 const (
 	// indentString represents the default indentation string used for
 	// JSON. Two spaces are used here.
-	indentString                 string = "  "
-	Default_REDIS_UNIXSOCKET     string = "/var/run/redis/redis.sock"
+	indentString				 string = "  "
```

Comment:
some unnecessary change. Could you please help to change them back? Thanks!

Reply:
I think this was removed by gofmt since it was vestigial.